kasigazibrian/WeConnectReact
# #157835150 Add coveralls

### What does this PR do?
Adds coveralls to get the test coverage

### Description of Task to be completed?
Add the coveralls.yml file with the

### How should this be manually tested?
Navigate to the repository and you should observe the coverage badge
<img width="316" alt="screen shot 2018-07-04 at 13 11 20" src="https://user-images.githubusercontent.com/26734746/42271523-39fe86be-7f8c-11e8-872d-5fef7383c0bc.png">


### Any background context you want to provide?
Previously, there was no coverage for the tests implemented.

What are the relevant pivotal tracker stories?
[#157835150](https://www.pivotaltracker.com/story/show/157835150)
